shared/constants/network.ts

```diff
@@ -782,6 +784,7 @@ export const CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP = {
   [CHAINLIST_CHAIN_IDS_MAP.BASE]: BASE_TOKEN_IMAGE_URL,
   [CHAINLIST_CHAIN_IDS_MAP.NUMBERS]: NUMBERS_MAINNET_IMAGE_URL,
   [CHAINLIST_CHAIN_IDS_MAP.SEI]: SEI_IMAGE_URL,
+  [CHAINLIST_CHAIN_IDS_MAP.APE_TESTNET]: APE_TESTNET_IMAGE_URL,
```

Comment:
[nit] from 4 items above, list is no longer alphabetically ordered. It would be best to maintain alphabetical order for better readability

Reply:
Dnt think the full list is sorted alphabetically but i can move this up before [CHAINLIST_CHAIN_IDS_MAP.BASE]